annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/AnnotationServiceImpl.java

```diff
@@ -646,6 +648,23 @@ private void validateTag(Annotation webAnnotation) throws ParamValidationExcepti
 	}
     }
 
+    /**
+     * Validation of transcribing.
+     * 
+     * @param webAnnotation
+     */
+    private void validateTranscribing(Annotation webAnnotation) throws ParamValidationException {
+		Body body = webAnnotation.getBody();
+		Target target = webAnnotation.getTarget();
+	
+		// the body type shouldn't be null at this stage
+		if (!(body instanceof SpecificResourceBody)) {
+		    throw new ParamValidationException(ParamValidationException.MESSAGE_WRONG_CLASS,
```

Comment:
move this validation into the validateTranscribingWithSpecificResource method

Reply:
moved into validateTranscriptionWithSpecificResource - also renamed this method similar like above